ledger/src/blockstore.rs

```diff
@@ -148,6 +148,52 @@ pub struct BlockstoreSignals {
     pub completed_slots_receiver: CompletedSlotsReceiver,
 }
 
+#[derive(Default)]
+pub struct TurbineFecSetStats(HashMap<Slot, HashMap</*fec_set_index*/ u32, /*count*/ usize>>);
+
+impl TurbineFecSetStats {
+    pub fn inc_index_count(&mut self, slot: Slot, fec_set_index: u32) {
+        *self
+            .0
+            .entry(slot)
+            .or_default()
+            .entry(fec_set_index)
+            .or_default() += 1;
+    }
+
+    pub fn get_min_index_count(&self, slot: &Slot) -> (usize, usize) {
+        let m = self.0.get(slot);
+        if m.is_none() {
+            return (0, 0);
+        }
+        let m = m.unwrap();
```

Comment:
Maybe use a more descriptive name here than just `m`.
Also `.is_none()` then `.unwrap()` is checking for none twice.
Instead, something like:
```rust
let stats = match self.0.get(slot) {
   None => return (0, 0),
   Some(stats) => stats,
};
```